### ISSUE TITLE
Bug: No such file or directory error during upgrade

### ISSUE THREAD

**Issue by joshknowles** [qltysh/qlty#1202]
### What happened?

qlty init asked me to upgrade
During upgrade, got error for "No such file or directory"

### What did you expect to happen?

Upgrade without errors

### Can you reproduce the problem?

Yes, consistently

### CLI Version

0.448

### Relevant log output

```shell
qlty init

A new version 0.448.0 of qlty is available!
✔ Do you want to upgrade qlty now? · yes

Running qlty upgrade --version 0.448.0 ...

❌ No such file or directory (os error 2)
› Initializing qlty in the current directory...
✔ Created .qlty/ directory
✔ Fetched plugins definitions
✔ Created .gitignore in .qlty/
✔ Created qlty.toml config file with 18 plugins
```

**Comment by marschattha** [comment]
I don't think this is happening during upgrade, probably happening afterwards, can you run `qlty --version` to verify that upgrade was done successfully or not?

**Comment by joshknowles** [comment]
> can you run qlty --version to verify that upgrade was done successfully or not?
`qlty 0.448.0 (4aede17)`

**Comment by brynary** [comment]
@joshknowles can you please run this command twice?:

```sh
qlty upgrade --force
```

The upgrade command runs the currently installed version, and then tries to bump to the latest release. Our current version is v052.0 and I am trying to figure out if this issue still exists when running a "from v0.520.0" upgrade or has been resolved between v0.448.0 and now.

**Comment by brynary** [comment]
Thanks! OK, so that all looks good.

So the upgrade system seems to be working fine as of the latest version.

The only thing that I can think of which this did not specifically exercise is the case of upgrading on init -- which was the example from your original screenshot.

We can keep an eye out for that to crop up again. In the meantime, we will close this out unless this occurs again or is reproducible by other means on your system.
